ocs_ci/ocs/ui/page_objects/block_and_file.py

```diff
@@ -0,0 +1,149 @@
+        """
+        Initial page - Data Foundation / Storage Systems tab / StorageSystem details / ocs-storagecluster-cephblockpool
+        Read the capacity breakdown from the table
+
+        Returns:
+            dict: Dictionary of capacity breakdown
+        """
+        logger.info("Read the capacity breakdown from the table")
+
+        num_capacity_labels = len(
+            self.get_elements(self.validation_loc["capacity_breakdown_cards"])
+        )
+        card_to_size = dict()
+        for i in range(1, num_capacity_labels + 1):
+            loc_card_name = format_locator(
+                self.validation_loc["capacity_breakdown_card"], str(i)
+            )
+            link = self.get_element_attribute(loc_card_name, "href").get("baseVal")
+            card_name = link.split("/")[-1]
+            loc_card_size = format_locator(
+                self.validation_loc["capacity_breakdown_card_size"], str(i)
+            )
+            title_and_size = self.get_element_text(loc_card_size)
+            # card name and shortening '...' removal
+            card_size = title_and_size.split("...")[-1]
+            card_to_size[card_name] = card_size
+
+        logger.info(card_to_size)
+        return card_to_size
+
+    def check_pvc_to_namespace_ui_card(self, namespace, check_name: str):
+        self.select_capacity_resource("PersistentVolumeClaims", namespace)
+        pvc_to_size_dict = self.read_capacity_breakdown()
+        self.take_screenshot()
+        if not all(
+            [
+                pvc_to_size_dict.get(pvc_name)
+                for pvc_name in PvcCapacityDeploymentList().get_pvc_names_list()
+            ]
+        ):
+            self.take_screenshot()
+            self.copy_dom()
+            return {
+                check_name: f"pvc's from UI: {PvcCapacityDeploymentList().get_pvc_names_list()} "
+                f"are not the same as expected pvc's: {pvc_to_size_dict}"
+            }
+
+        for data_struct in PvcCapacityDeploymentList():
+
+            used_capacity_ui: str = pvc_to_size_dict[data_struct.pvc_obj.name]
+            used_capacity_expected_int = data_struct.capacity_size
+
+            if used_capacity_expected_int is None:
+                self.take_screenshot()
+                self.copy_dom()
+                return {
+                    check_name: f"Failed to extract a number from the value given at UI "
+                    f"for the pvc: {data_struct.pvc_obj.name}"
+                }
+
+            if not compare_mem_usage(
+                used_capacity_expected_int, used_capacity_ui, deviation_accepted=10
+            ):
+                self.take_screenshot()
+                self.copy_dom()
+                return {
+                    check_name: f"pvc {data_struct.pvc_obj.name} capacity is not as expected. "
+                    f"cli capacity: {data_struct.capacity_size}GiB | ui capacity {used_capacity_ui}"
+                }
+            else:
```

Comment:
in this case, the function doesn't return a value. This contradicts with the docstring info

Reply:
If the function doesn't have any return statement, then it returns None. 

Regarding the method itself, it expected to have None after the check. check_pvc_to_namespace_ui_card will return only information about failed data comparisons, so we can validate them in the end of the test.  This is an implementation of what I called a complex test scenario - "do not fail the test on first bad occasion but write it somewhere and check in the end of the test".

Also I don't have a docstring. I will add it right now. 
Thanks